profile/README.md

```diff
@@ -1,3 +1,30 @@
 # California Cropland Monitoring and Modeling Framework
 
 ## About
+
+The California Cropland Monitoring and Modeling Framework (CCMMF) will be a biogeochemical model and data integration pipeline to generate inventories and future projections of soil carbon (SOC) stocks and greenhouse gas fluxes (GHG) from California’s croplands. 
+CCMMF will leverage Bayesian techniques to combine heterogeneous datasets into unified wall-to-wall inventories and climate and scenario-based projections. 
+Data layers will be generated as ensemble projections to facilitate propagation of uncertainty through downstream applications.
```

Comment:
"The data layers generated in our model-data integration framework..." "..though downstream applications, including those designed to inform land management and climate policy"?